std_msgs/Header header
sensor_msgs/PointCloud2[] clusters

sensor_msgs/PointCloud2[] surfaces
